app/src/main/res/values/dimens.xml

```diff
@@ -133,4 +164,5 @@
     <dimen name="progress_bar_height">8dp</dimen>
     <dimen name="connect_instruction_text_header">18sp</dimen>
 
-</resources>
```

Comment:
Also revert changes done here at the end of the file. It's safe not to do any modifications to the end of resource files.